src/ldc/eh/common.d

```diff
@@ -287,12 +287,72 @@ void _d_getLanguageSpecificTables(ubyte* data, ref ubyte* callsite, ref ubyte* a
     debug(EH_personality) printf("  - callsite: %p, action: %p, classinfo_table: %p, ciEncoding: %d\n", callsite, action, classinfo_table, ciEncoding);
 }
 
+// ---------------------------------------------------------------
+//    A fixed pool of T instances, used to optimise allocations
+//    Uses a free list to ensure fast de/allocation
+// ---------------------------------------------------------------
+template FixedPool(T, int N)
```

Comment:
There's no major reason behind not using struct; I felt like it'd be more idiomatic to use a template block so that marking the methods static would be unnecessary. Happy to change it.

As for moving it to another module / unit tests: I actually had it in another module (with unit tests) to begin with, but I didn't want to introduce too much new code. Also a fairly easy fix (I'd just move it to `core.internal` or the like.)